sessions/js-functions-2/vending-machine_advanced/js/index.js

```diff
@@ -0,0 +1,31 @@
+console.clear();
+
+/* --- || Vending Machine || --- */
+/* --- --- --- */
+/* --- --- --- */
+/* --- || Goal: Write a fat arrow function/ arrow function expression that adds some additional features to our basic vending machine. - */
+
+/* --- ! Read all of the tasks carefully before starting to code ! --- */
+
+/* 1. Remember the "vendingMachine" function you've created as part of the Functions 1 challenges?
+Now write the same function still called "vendingMachine", but as an arrow function expression. */
+
+/* 2. The function should include one parameter: "money". */
+
+/* 3. 
+In the function body, implement the following logic:
+- an early return for the condition "money === 0"
+- an if condition each (!) for "money === 1", "money === 2" and  "money === 3" that
+  log different strings to the console, such as "Here's your water!", "Here's your juice!", etc.
+- another if condition for when "money" either equals "4" or anything above that logs "Too much money!"
+  to the console.
+
+  The behavior we're looking for is to get different (or no) drinks each when we 
+  pass the numbers 0, 1, 2, 3 and 4+ to the "vendingMachine" function as an argument.
```

Comment:
lets move this part to the top of the task description, so the students know what they should build.